spring-aop/src/test/java/org/springframework/aop/aspectj/annotation/AbstractAspectJAdvisorFactoryTests.java

```diff
@@ -170,7 +171,7 @@ public void testMultiplePerTargetAspectsWithOrderAnnotation() throws SecurityExc
 		PerTargetAspectWithOrderAnnotation5 aspect2 = new PerTargetAspectWithOrderAnnotation5();
 		advisors.addAll(
 				getFixture().getAdvisors(new SingletonMetadataAwareAspectInstanceFactory(aspect2, "someBean2")));
-		Collections.sort(advisors, new OrderComparator());
+		sort(advisors, new OrderComparator());
```

Comment:
If optimize, then do one step further :)

> sort(advisors, OrderComparator.INSTANCE);

Reply:
Hello, I am modifying it to OrderComparator.sort(advisors);